templates/roles/gpu/tasks/main.yml

```
@@ -8,31 +8,13 @@
   apt_repository:
     repo: "deb http://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64 /"
 
-- name: Install CUDA 11.4
+- name: Install CUDA 11.5
```

```suggestion
- name: Install CUDA
```

I think since we've moved the version numbers to variables, we can just call this `Install CUDA`.

---

I agree